contrib/clemux/docker/debile-slave/Dockerfile

```diff
@@ -15,7 +15,16 @@ COPY *.deb /tmp/
 
 WORKDIR /tmp
 
-RUN apt-get update && apt-get install -y python python-dput dput-ng python-firehose adduser python-debian python-requests python-yaml sbuild python-schroot
+RUN apt-get update && apt-get install -y python python-dput dput-ng python-firehose adduser python-debian python-requests python-yaml python-schroot wget
+
+# Use a custom version of sbuild written by Léo Cavaillé that adds an
+# option --use-schroot-session, which enables starting sbuild with an
+# existing schroot session.
+RUN wget https://dimjasevic.net/marko/debile/libsbuild-perl_0.69.0-2_all.deb
```

Comment:
Having sbuild built from source such that the patch is applied to it sounds great! I'll do that in the Dockerfile.